README.md

```diff
@@ -1 +1,34 @@
-# PortofolioSetup1
\ No newline at end of file
+# PortofolioSetup1
+## Project Name
+> This is my firest part of a continues website development project which depend on a very good website calld figma.
```

Comment:
- [ ] Please try to correct the spelling mistakes. We can use [Grammarly](https://app.grammarly.com/) for avoiding these mistakes.